node/src/main/scala/coop/rchain/node/service/jsonrpc.scala

```diff
@@ -0,0 +1,128 @@
+package coop.rchain.node.service
+
+import cats.effect._
+import io.circe._
+import io.circe.generic.auto._
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras._
+import io.circe.literal._
+import io.circe.syntax._
+import org.http4s._
+import org.http4s.circe._
+import org.http4s.dsl.io._
+
+import shapeless._
+
+object JsonRpc {
```

Comment:
It's customary to have this match the filename in the case of a single object/class per file.  If there is more than a single object/class, then the convention is to use lowercase.